ee/control/bucket_consumer.go

```diff
@@ -28,6 +28,10 @@ func NewBucketConsumer(logger log.Logger, db *bbolt.DB, bucketName string) *buck
 }
 
 func (bc *bucketConsumer) Update(data io.Reader) error {
+	if bc == nil {
```

Comment:
Only in testing code. Since some objects can use `bucketConsumer` as a sort of cache, allowing it to be nil gracefully disables the cache, since any cache errors are usually treated as non-fatal.